client/src/pages/index.tsx

```diff
@@ -10,20 +10,19 @@ import { CircularProgress } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import Cmi5 from "@xapi/cmi5";
 import { hasCmi } from "cmiutils";
-import Chat from "components/chat";
 import GuestPrompt from "components/guest-prompt";
 import Header from "components/header";
 import Input from "components/input";
 import Video from "components/video";
 import VideoPanel from "components/video-panel";
 import { loadConfig, loadMentors, setGuestName } from "store/actions";
-import { Config, LoadStatus, MentorType, State } from "types";
+import { ChatProps, Config, LoadStatus, MentorType, State } from "types";
 import withLocation from "wrap-with-location";
 import "styles/layout.css";
 import { fetchMentorByAccessToken } from "api";
 import { createMuiTheme, MuiThemeProvider } from "@material-ui/core/styles";
 
-import HistoryChat from "components/history";
+import Chat from "components/chat";
```

Comment:
this is RE the relative import below... can we not make that an abs import (get rid of the `..`)?

Reply:
this is not resolved. Can we not fix the relative import below?

Reply:
This is still not resolved. Please fix the line

```typescript
// import "../styles/history-chat-responsive.css";
import "styles/history-chat-responsive.css";
```

...if it can't be fixed, as I'm asking for some reason, then explain why in a response comment, and then I will resolve the change request if I agree.

Please DO NOT mark anything `resolved` in a PR review unless the requested change is clear, and you have actually made the change.

Reply:
fixed